doc/usecases/layers/zones_layer.md

```diff
@@ -0,0 +1,23 @@
+# Use Case: Zones Layer
+
+## Summary
+
+- **Scope:** Zones Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can add, edit, move and delete zones in their plan using the zones layer. Zones can be added via common shapes or hand drawn polygons.
```

Comment:
:+1: This is a good idea for zones layers. Please also add brushing functionality, like for other layers.

In zones the zones exclude each other (different to animals, where they can overlap).

Prefer to keep **Brief:** short but details (like how Zones are drawn) to **Main success scenario:**